### ISSUE TITLE
Connected sortableInput: hidden filed is not updated after drag and drop

### ISSUE THREAD

**Issue by bobonov** [kartik-v/yii2-sortable-input#1]
After drag and drop the filed sometime is not updated.
I do not javascript error, it simply seem that the update action sometime does not run after successful drop event.
My code snippet:

``` php
<div class="row">
    <div class="col-md-6">
        <?=
        SortableInput::widget([
            'name' => 'origine',
            'id' => 'origine',
            'sortableOptions' => [
                'connected' => true,
                'showHandle' => true,
                'type' => 'grid',
                'itemOptions' => ['style' => 'max-width: 48%;'],
            ],
            'hideInput' => false,
            'items' => $item
        ]);
        ?>
    </div>
    <div class="col-md-6">
        <?=
        SortableInput::widget([
            'name' => 'destinazione',
            'id' => 'destinazione',
            'sortableOptions' => [
                'connected' => true,
                'showHandle' => true,
                'type' => 'grid',
                'itemOptions' => ['style' => 'max-width: 48%;'],
            ],
            'hideInput' => false,
            'items' => [],
        ]);
        ?>
    </div>
</div>
```

**Comment by it3rmit** [comment]
+1
I have the same issue. I use a few connected sortable-inputs. When I drag and drop an item from one list to another, the input field in updated in the last list and isn't updated in the first one(which has lost an item). My code:

``` php
        echo SortableInput::widget([
                        'name'=> 'sort_list_1',
                        'value'=>'3,4,2,1',
                        'items' => [
                            1 => ['content' => 'Item # 1'],
                            2 => ['content' => 'Item # 2'],
                            3 => ['content' => 'Item # 3'],
                            4 => ['content' => 'Item # 4'],
                        ],
                        'hideInput' => false,
                        'sortableOptions' => [
                            'connected' => true,
                            'itemOptions'=>['class'=>'alert alert-success'],
                            'disabled' => false
                        ],
                        'options' => ['class'=>'form-control'],
                        //'hideInput' => true
                    ]);

        echo SortableInput::widget([
                    'name'=> 'sort_list_2',
                    'value'=>'6,5,8,7',
                    'items' => [
                        5 => ['content' => 'Item # 5'],
                        6 => ['content' => 'Item # 6'],
                        7 => ['content' => 'Item # 7'],
                        8 => ['content' => 'Item # 8'],
                    ],
                    'hideInput' => false,
                    'sortableOptions' => [
                        'connected' => true,
                        'itemOptions'=>['class'=>'alert alert-warning'],
                        'disabled' => false
                    ],
                    'options' => ['class'=>'form-control'],
                    //'hideInput' => true
                ]);
```

**Comment by kartik-v** [comment]
This needs upgrade to the source plugin using a fork... will update.

**Comment by kartik-v** [comment]
Resolved via upgrade to release v1.2.0.

**Comment by kartik-v** [comment]
Check the [scenario # 5 in USAGE section on the demos page](http://demos.krajee.com/sortable-input) for demo on connected sortable inputs.
